test/utils/test_Utils.py

```diff
@@ -0,0 +1,40 @@
+
+import unittest
+from unittest.mock import patch 
+from utils.utils import is_int, try_get_int_value
+
+class TestMainFunctions(unittest.TestCase):
+    def test_is_int(self):
+        self.assertTrue(is_int("5"))
+        self.assertTrue(is_int("-3"))
+        self.assertTrue(is_int("0"))
+        self.assertFalse(is_int("10.0"))
+        self.assertFalse(is_int("str"))
+        self.assertFalse(is_int("4t"))
```

Comment:
Instead of making many assertions in the test, you could use a simple test with @paramtetrized decorator:
https://github.com/wolever/parameterized

in which you define list of inputs and expected outputs, and the test is run multiple times under the hood